packages/validation-manager/src/GethTracer.ts

```diff
@@ -26,16 +27,17 @@ type LogTracerFunc = () => LogTracer
  * @param provider the network node to trace on
  * @param tx the transaction to trace
  * @param options the trace options
- * @param nativeTracerProvider if set, submit only preStateTracer to the network provider, and use this (second) provider with native tracer.
+ * @param prestateTracerProvider if set, submit only preStateTracer to the network provider, and use this (second) provider with native tracer.
```

Comment:
The comment is not very clear or correct.

Reply:
This is going to be removed completely